### PR TITLE
feat: 3ds Max integration

### PR DESCRIPTION
**Pull request by henriknorin-ftrack**
Resolves 

- [X] The PR contains updates to the release notes.

## Changes

3ds Max integration

## Test

_Note: needs to be tested with corresponding story  branch on pipeline, definition, qt and 3dsmax branches._

Run a full cycle 3ds Max test.

### CODE REVIEW

**Review by lluisCM — APPROVED** [review]
LGTM

File: doc/release/release_notes.rst [hunk 1]
```diff
@@ -7,6 +7,13 @@
 Release Notes
 *************
 
+.. release:: upcoming
+
+    .. change:: new
+
+    3ds Max integration.
```
Comment (hdd) [inline doc/release/release_notes.rst]: @henriknorin-ftrack 

something more like: 

`Add 3dsmax definition and plugins `

might explain better what is this about

File: resource/definitions/loader/3dsmax/rig-3dsmax-loader.json [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+    },
+    {
+      "name": "model",
+      "file_formats": [".max"],
+      "stages": [
+        {
+          "name": "collector",
+          "plugins":[
+            {
+              "name": "Collect components from context",
```
Comment (hdd) [inline resource/definitions/loader/3dsmax/rig-3dsmax-loader.json]: @henriknorin-ftrack 

`Collect components from context`

is it what does this do ?

Comment (henriknorin-ftrack) [reply resource/definitions/loader/3dsmax/rig-3dsmax-loader.json]: # Plugin that collects loader compatible component paths

I'll change to something like this

File: doc/release/release_notes.rst (hunk 1 above)
Comment (henriknorin-ftrack) [reply doc/release/release_notes.rst]: Changing

**Pull request merged**